content/logs/faq/setting-file-permissions-for-rotating-logs.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Setting file permissions for rotating logs (linux)
+kind: faq
+further_reading:
+- link: "/logs/faq/log-parsing-best-practice"
+  tag: "FAQ"
+  text: Log Parsing - Best Practice
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+The datadog agent runs under the dd-agent user and dd-agent group. This prevents dd-agent accessing the logs in `/var/log` as they are only accessible by root (or a sudo admin).
+
+## Setting permissions using ACL's
+
+In order to allow read only access for dd-agent only, [create ACL's and modify logrotate to persist the permissions changes](https://help.ubuntu.com/community/FilePermissionsACLs).
```

Comment:
We should document this in a more generic way, to ensure we cover all supported platform.s

Reply:
I agree this is the first step, other PR will follow to support other platforms.

Reply:
s/datadog agent/Datadog Agent/
I'd put dd-agent for both user and group in backticks, so `dd-agent` and clarify `This prevents the Datadog Agent from accessing the logs...`

s/read only/read-only/

Reply:
👌